app/views/transactions/new.html.erb

```diff
@@ -0,0 +1,2 @@
+
+<%= render "form", transaction: @transaction %>
```

Comment:
- [ ] Please kindly display great validation messages to the user for a better experience. A user should be aware of why he is unable to create a transaction through these great messages.